h2o-py/h2o/model/model_base.py

```diff
@@ -1088,6 +1088,8 @@ def partial_plot(self, data, cols=None, destination_key=None, nbins=20, weight_c
         :param targets: Target classes for multiclass model.
         :returns: Plot and list of calculated mean response tables for each feature requested.
         """
+        if save_to_file is None:  # if not supplied by 'real name'
+            save_to_file = kwargs.get('save_plot_path')
```

Comment:
if the idea is to deprecate `save_to_file` param for consistency (which I approve), we have a decorator for this:
```python
@deprecated_params({'save_to_file': 'save_plot_path'})
def partial_plot(self, data, …, save_plot_path=None, …):
    …
```
for new code, it will promote `save_plot_path` (as `save_to_file` won't appear anymore in the list of params, nor in the doc).
for legacy code, it will translate on the fly `save_to_file` param into `save_to_path`, which I believe is what we want to achieve.

Reply:
Thanks! be2b8e3